editor/styling/stylesheets/_editor.scss

```diff
@@ -67,7 +67,7 @@ Links:
   -fx-text-color-base: -df-defold-blue-lighter;
   -fx-text-fill: -df-defold-blue-lighter; }
 
-&.override-inspector-check-box {
+.override-inspector-check-box {
```

Comment:
There was a top-level `&`, which is wrong since `&` refers to a parent, and on the top level no parent exists. Dart Sass reports it as error.